### ISSUE TITLE
OverlayItem - Item not existing shawn always in right-bottom corber

### ISSUE THREAD

**Issue by Fausto70** [osmdroid/osmdroid#1444]
## Issue Type

[ ] Question
[X ] Bug
[ ] Improvement
[ ] Build system related
[ ] Performance
[ ] Documentation

## Description and/or steps/code to reproduce the problem
I upgraded my app to the last OSMdroid (OSMdroid 6.1.4 - Bonuspack 6.6.0)
Before I had OSMdroid 5.1 and bonus pack 5.8.1

After upgrading I have a strange bug. 
let's assume I have to show 1 item on the map:, by the following code:

```
OverlayItem MY_OverlayItem = new OverlayItem("1", "LABEL", "", myGeoPoint);
MY_OverlayItem.setMarker(myIcon);

ArrayList<OverlayItem> ARRAY_Of_OverlayItems = new ArrayList<>(); 
ARRAY_Of_OverlayItems.add(MY_OverlayItem);

ItemizedOverlayWithFocus myItemizedOverlayWithFocus = new ItemizedOverlayWithFocus<>(this, ARRAY_Of_OverlayItems__Arg,  new ItemizedIconOverlay.OnItemGestureListener<OverlayItem>(){
.... some listeners .....
}

myItemizedOverlayWithFocus.setFocusItemsOnTap(true);
myItemizedOverlayWithFocus.setFocusedItem(0);
myMapView.getOverlays().add(myItemizedOverlayWithFocus);
myMapView.invalidate();

```

I see an item on the correct position on the map, but without the label.
Then I see on the right-bottom corner of the Android screen another item, with the same icon and with the label "LABEL".
If I change the zoom, the wrong item remains always in the right-botton corner of the screen.

### If it's a bug, version(s) of android this affects:
All

### Version of osmdroid the issue relates to: 
6.1.4

**Comment by spyhunter99** [comment]
confirmed bug. looks like it's related to setting an explicit icon. i was able to reproduce with SampleMilitaryIconsItemizedIcons and with your code.

**Comment by spyhunter99** [comment]
#944 #990 #1322 possible causes

**Comment by spyhunter99** [comment]
in the mean time, i'd suggest you use the marker class instead

**Comment by monsieurtanuki** [comment]
@spyhunter99 I've just fixed something (#1445) that might be related.
Should we deprecate `ItemizedOverlay` and always use the (supposedly) more powerful `Marker` instead? I thing I remember that @MKergall also suggested something like that.

**Comment by Fausto70** [comment]
Thank you very much... can you suggest a link where I can find some example of use Marker class instead of ItemizedOverlay  ?
My target is to show some items on the map, and have some listeners active when the user tap on the item.

ThankYou in Advance

**Comment by monsieurtanuki** [comment]
@Fausto70 perhaps https://github.com/osmdroid/osmdroid/blob/master/OpenStreetMapViewer/src/main/java/org/osmdroid/samplefragments/data/SampleMarker.java

**Comment by spyhunter99** [comment]
> 
> 
> @spyhunter99 I've just fixed something (#1445) that might be related.
> Should we deprecate `ItemizedOverlay` and always use the (supposedly) more powerful `Marker` instead? I thing I remember that @MKergall also suggested something like that.

agreed. your fix resolves the issue. i can cut another release this weekend

**Comment by Fausto70** [comment]
I solved at the moment, as suggested by Spyhunter, using Marker class.
But at the moment the icon size is very small. I was searching for a way to change the icon size, but I didn't find any method on Marker object. Any suggestion?
